vendor/inline-scripts/pageview-tracking-experiment.js

```diff
@@ -0,0 +1,104 @@
+(function pageViewTracking(config = {}) {
+  let beacon;
+  let sessionId;
+  let pvNumber;
+  let pvNumberGlobal;
+  let headData;
+
+  function readCookies() {
+    beacon = getCookieValue('wikia_beacon_id');
+    sessionId = getCookieValue('tracking_session_id') || genUID();
+    pvNumber = getCookieValue('pv_number');
+    pvNumberGlobal = getCookieValue('pv_number_global');
+  }
+
+  function getCookieValue(cookieName) {
+    const cookieSplit = ('; ' + document.cookie).split('; ' + cookieName + '=');
+
+    return cookieSplit.length === 2 ? cookieSplit.pop().split(';').shift() : null;
+  }
+
+  function genUID() {
+    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
+      const r = Math.random() * 16 | 0, v = c === 'x' ? r : (r & 0x3 | 0x8);
+
+      return v.toString(16);
+    });
+  }
+
+  function getEventParams(shouldIncreasePvNumber) {
+    let pvNumberComputed = pvNumber ? parseInt(pvNumber, 10) : 0;
+    let pvNumberGlobalComputed = pvNumberGlobal ? parseInt(pvNumberGlobal, 10) : 0;
+    if (shouldIncreasePvNumber) {
```

Comment:
for initial page view `shouldIncreasePvNumber` is true as the FCP tracking event is sent before the regular page view event. For the next SPA page views the event is run after the regular page view event so the value is already increased.